docs/_data/team.js

```diff
@@ -0,0 +1,152 @@
+    name: 'Shinigami',
+    title: 'Maintainer',
+    org: 'Faker',
+    orgLink: 'https://fakerjs.dev',
+    desc: 'Passionate TypeScript enthusiast working extensively with Vue SPA and pug.',
+    links: [
+      { icon: 'github', link: 'https://github.com/Shinigami92' },
+      { icon: 'twitter', link: 'https://twitter.com/Shini_92' }
+    ],
+    sponsor: 'https://github.com/sponsors/Shinigami92'
+  },
+  {
+    avatar: 'https://github.com/aleclarson.png',
+    name: 'aleclarson',
+    title: 'Open sourceror',
+    desc: 'Risk taker, Free thinker, App builder, Board game believer.',
+    links: [
+      { icon: 'github', link: 'https://github.com/aleclarson' },
+      { icon: 'twitter', link: 'https://twitter.com/retropragma' }
+    ],
+    sponsor: 'https://github.com/sponsors/aleclarson'
+  },
+  {
+    avatar: 'https://github.com/bluwy.png',
+    name: 'Bjorn Lu',
+    title: 'Frontend Developer',
+    org: 'Opslock',
+    orgLink: 'https://opslock.com',
+    desc: 'Svelte and Vite team member. Something something opinions.',
+    links: [
+      { icon: 'github', link: 'https://github.com/bluwy' },
+      { icon: 'twitter', link: 'https://twitter.com/bluwyoo' }
+    ],
+    sponsor: 'https://bjornlu.com/sponsor'
+  },
+  {
+    avatar: 'https://github.com/poyoho.png',
+    name: 'yoho',
+    title: 'Frontend Developer',
+    desc: 'Frontend. Vite team member.',
+    links: [
+      { icon: 'github', link: 'https://github.com/poyoho' },
+      { icon: 'twitter', link: 'https://twitter.com/yoho_po' }
+    ]
+  },
+  {
+    avatar: 'https://github.com/sapphi-red.png',
+    name: 'green',
+    title: 'Web Developer',
+    desc: 'Vite team member. Call me sapphi or green or midori ;)',
+    links: [
+      { icon: 'github', link: 'https://github.com/sapphi-red' },
+      { icon: 'twitter', link: 'https://twitter.com/sapphi_red' }
+    ]
+  },
+  {
+    avatar: 'https://github.com/ygj6.png',
+    name: 'ygj6',
+    title: 'Developer',
+    desc: '...?',
+    links: [{ icon: 'github', link: 'https://github.com/ygj6' }]
+  },
+  {
+    avatar: 'https://github.com/Niputi.png',
+    name: 'Niputi',
+    title: 'Developer',
+    org: 'Computershare Denmark',
+    desc: 'weeb/javascript lover.',
+    links: [
+      { icon: 'github', link: 'https://github.com/Niputi' },
+      { icon: 'twitter', link: 'https://twitter.com/Niputi_' }
+    ]
+  }
+]
+
+export const emeriti = [
+  {
+    avatar: 'https://github.com/underfin.png',
+    name: 'underfin',
+    title: 'Developer',
+    links: [{ icon: 'github', link: 'https://github.com/underfin' }]
+  },
+  {
+    avatar: 'https://github.com/GrygrFlzr.png',
+    name: 'GrygrFlzr',
+    title: 'Developer',
+    links: [{ icon: 'github', link: 'https://github.com/GrygrFlzr' }]
+  },
+  {
+    avatar: 'https://github.com/nihalgonsalves.png',
+    name: 'Nihal Gonsalves',
+    title: 'Senior Software Engineer',
+    org: 'Delivery Hero',
+    orgLink: 'https://www.deliveryhero.com',
```

Comment:
Nihal requested to remove this info
```suggestion
    org: 'Delivery Hero',
    orgLink: 'https://www.deliveryhero.com',
```